index.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const helmet = require('helmet');
+const server = require('./server.js');
+const cohortRoute = require('./routes/cohortsRoute.js')
+
+server.use(express.json());
+server.use(helmet());
+
+server.use('/api/cohorts', cohortRoute);
+
+server.get('/', async (req, res) => {
+    try {
+        res.status(200).json({message: "Welcome to database."})
+    } catch (error) {
+        res.status(500).json(error)
+    }
+});
+
+server.listen(5000, () => {
+    console.log("\n *** Running on port 5000 *** \n")
+});
```

Comment:
I would recommend, instead of making a server.js file to export, to just create an instance of server inside this file, since your server.js file doesn't do anything like routes